Meta tweak (#1337)
leachim6/hello-world: .github/pull_request_template.md

## Adding a language

> - [ ] The code displays "Hello World" (you can check on [tio.run](https://tio.run) if the language is available)

Reviewer: ```suggestion
- [ ] The code displays "Hello World" ([tio.run](https://tio.run/) may help for testing)
```
